README.md

```diff
@@ -0,0 +1,67 @@
+
+# Overview
+
+Terraform module for setting up infrastructure for [Shunting Yard](https://github.com/HotelsDotCom/shunting-yard).
+
+For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
+
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| aws_region | AWS region to use for resources. | string | - | yes |
+| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
+| memory | The amount of memory (in MiB) used to allocate for the Shunting Yard container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
+| sy_ecs_task_count | Number of ECS tasks to create. | string | `1` | no |
+| docker_image | Full path to Shunting Yard Docker image. | string | - | yes |
+| docker_version | Shunting Yard Docker image version. | string | - | yes |
+| ingress_cidr | Generally allowed ingress CIDR list. | list | - | yes |
+| instance_name | Shunting Yard instance name to identify resources in multi-instance deployments. | string | `` | no |
+| subnets | ECS container subnets. | list | - | yes |
+| tags | A map of tags to apply to resources. | map | `<map>` | no |
+| vpc_id | VPC ID. | string | - | yes |
+| source_metastore_uri | Source Metastore URI to be configured in Shunting Yard | string | `localhost` | yes |
+| target_metastore_uri | Target Metastore URI to be configured in Shunting Yard | string | `localhost` | yes |
```

Comment:
no `default` set for this variable in `variables.tf`

Reply:
Ah ! Nice catch. I have fixed this in all places.